react/README.md

```diff
@@ -11,34 +11,58 @@ Mayflower-react is a React component library under the [enterprise design system
 
 
 ## Using Mayflower-React in Your Project
-1. Install mayflower-react into your project as a dependency.
-`npm i @massds/mayflower-react --save`
-2. Import components into your App.js
-`import { Button } from '@massds/mayflower-react';`
-3. Render components in JSX
-`<Button text="Button" onClick={() => console.log('mayflower button clicked!')} />`
-
-Please note that [@massds/mayflower-react][npm] comes with prebuilt css, svg and other image files. If you are configuring your own Webpack, you might need to add the following loaders to handle these files:
-``npm install --save-dev css-loader svg-inline-loader file-loader``
-
-You can also kickstart your React project with Mayflower-react referring to the [Mayflower React Starter][react-starter] built with [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
-
-For a guide and information on the components included in mayflower-react and their functionality, visit our [Mayflower React Storybook][react-storybook]. Click on the Info and Knobs tabs for component prop types, details and options.
+1. Install mayflower-react and mayflower-assets into your project dependency:
+```javascript
+npm i @massds/mayflower-react @massds/mayflower-assets --save
+```
+2. Import components into your App:
+```javascript
+import { Header, Footer, Button } from '@massds/mayflower-react';
+// You can also directly point to each component's path:
+import Header from '@massds/mayflower-react/dist/Header';
```

Comment:
```suggestion
import Header from '@massds/mayflower-react/dist/Header/index.mjs';
```

Reply:
ES Modules is explained further down. This change shouldn't happen.

Reply:
hmm ok, Can you move this section https://github.com/massgov/mayflower/pull/1109/files#diff-4f8f09a7b7e325026d10475db1bed44bR43-R53  up here to explain the imports based on project ES5/6 settings